src/Assets/Styles/components/_toggle.scss

```diff
@@ -0,0 +1,45 @@
+.js-toggle {
+  @include govuk-font($size: 19, $weight: 'bold');
+  @include govuk-responsive-margin(6, "bottom");
+  background-color: govuk-colour("grey-3");
+  background-image: url('/images/option-select-toggle-sprite.png');
+  background-position: right 5px;
+  background-repeat: no-repeat;
+  border: none;
+  color: $govuk-text-colour;
+  display: none;
+  padding: 8px;
+  text-align: left;
+  text-decoration: none;
+  width: 100%;
+
+  &.is-toggled {
+    background-position: right -75px;
```

Comment:
The arrow should be the other way around, the down arrow should show when the filter list is closed. This is what GOV.UK does: https://www.gov.uk/search?q=test

![screen shot 2018-08-06 at 15 30 16](https://user-images.githubusercontent.com/319055/43722643-a6b18d40-998d-11e8-9ede-9983f473b622.png)